### PR TITLE
Change clc++ script to search in lib64 for libraries

### PR DESCRIPTION
**Pull request by codekrafter**
Works properly:
- [x] x86_64

This changes the clc++ script (only when building from source) to include the lib64 folder. This fixes a bug when using clang++. This will also probably have to be fixed on bin-tray as well.

### CODE REVIEW

**Comment by JL2210** [comment]
You need to rebase against the master tree before creating PRs. I'm pretty sure glfw doesn't have anything to do with clang++.

File: packages/llvm.rb [hunk 1]
```diff
@@ -283,7 +283,8 @@ def self.build
 cxx_sys=#{CREW_PREFIX}/include/c++/#{gcc_ver}
 cxx_inc=#{CREW_PREFIX}/include/c++/#{gcc_ver}/x86_64-cros-linux-gnu
 gnuc_lib=#{CREW_PREFIX}/lib/gcc/x86_64-cros-linux-gnu/#{gcc_ver}
-clang++ -cxx-isystem \${cxx_sys} -I \${cxx_inc} -B \${gnuc_lib} -L \${gnuc_lib} \"\$@\"' > clc++"
+gnuc_lib64=#{CREW_PREFIX}/lib64/gcc/x86_64-cros-linux-gnu/#{gcc_ver}
+clang++ -cxx-isystem \${cxx_sys} -I \${cxx_inc} -B \${gnuc_lib} -B \${gnuc_lib654} -L \${gnuc_lib} -L \${gnuc_lib64} \"\$@\"' > clc++"
```
Comment (JL2210) [inline packages/llvm.rb]: Try gnuc_lib64 instead of gnuc_lib654?

File: packages/llvm.rb [hunk 2]
```diff
@@ -283,7 +283,8 @@ def self.build
 cxx_sys=#{CREW_PREFIX}/include/c++/#{gcc_ver}
 cxx_inc=#{CREW_PREFIX}/include/c++/#{gcc_ver}/x86_64-cros-linux-gnu
 gnuc_lib=#{CREW_PREFIX}/lib/gcc/x86_64-cros-linux-gnu/#{gcc_ver}
```
Comment (uberhacker) [inline packages/llvm.rb]: Better fix? `gnuc_lib=#{CREW_LIB_PREFIX}/gcc/x86_64-cros-linux-gnu/#{gcc_ver}`

**Comment by uberhacker** [comment]
Thank you for the fix @codekrafter!   No need to rebase.  This PR is fine.

**Comment by JL2210** [comment]
@uberhacker: As soon as your PR on updating GLFW is merged, this one will become invalid.

**Comment by JL2210** [comment]
Or the other way around. I always make sure to `git pull upstream master && git push origin master` before I create a PR.

**Comment by JL2210** [comment]
@codekrafter If you don't mind, I could fix this.

File: packages/llvm.rb (hunk 2 above)
Comment (JL2210) [reply packages/llvm.rb]: Yeah. I agree. Otherwise, I suppose this is fine.

**Pull request merged**